apps/system/js/bluetooth_transfer.js

```diff
@@ -483,6 +482,7 @@ var BluetoothTransfer = {
       if (transferInfo.received) {
         // Received file can be opened only
         nData.titleL10n = 'transferFinished-receivedSuccessful-title';
+        nData.contentType = transferInfo.contentType;
```

Comment:
To preview file (`openReceivedFile`) we need contentType(optional), so we will save it and send as `notification.data.contentType`